graph/patterns/enums.md

```diff
@@ -1,4 +1,4 @@
-### Enums
+# Enums

```

Comment:
@XVincentX could you please use the pattern template https://github.com/microsoft/api-guidelines/blob/vNext/graph/patterns/PatternDescriptionTemplate.md? Describing 1. what problem we are solving by using enums, 2. solution aka modeling enums, 3. when this solution is better than other patterns and 4. usage examples. The idea is to keep all articles in the same format. Google has a similar approach to AIPs https://google.aip.dev/8